omnibus/config/software/sds.rb

```diff
@@ -1,14 +1,17 @@
 name "sds"
 
-default_version "042de62f5a24fbceb4f4849256c3ee5c005b7057"
+default_version "v0.1.0"
 source git: 'https://github.com/DataDog/dd-sensitive-data-scanner'
 
 build do
     license "Apache-2.0"
     license_file "./LICENSE"
 
+    sds_version = "v0.1.0"
+
     # no Windows support for now.
     if linux_target? || osx_target?
+        command "git checkout #{sds_version}"
```

Comment:
I'm not 100% sure but think this isn't required. Omnibus should interpret the default version as a git ref and checkout automatically

Reply:
I'm giving it a shot!

Reply:
Seems to be working. Any chance you would know how to double-check this from the jobs worker or similar?